### DISCUSSION TITLE
import "@chainlink/..." raise not found but it still compile fine

### DISCUSSION THREAD

**Discussion by ofirbenatar** [smartcontractkit/full-blockchain-solidity-course-py#1068] [Q&A] [answered]
starting lesson five i had the following problem as this guy:
https://github.com/smartcontractkit/full-blockchain-solidity-course-py/discussions/865
compiling result in the following error:
`ParserError: Source "@chainlink/contracts/src/v0.8/interfaces/AggregatorV3Interface.sol" not found: File not found.
 --> contracts/Lottery.sol:5:1:
  |
5 | import "@chainlink/contracts/src/v0.8/interfaces/AggregatorV3Interface.sol";
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^`
i tried a bunch a different stuff and solve it by setting `smartcontractkit/chainlink-brownie-contracts@` to latest version 0.4.0 and not 1.1.1 like the course told. now its compiling fine but vscode still yell at me. i have sol compiler  version 0.6.6 and vscode global compiler is 0.6.6

<img width="877" alt="Screen Shot 2022-02-15 at 15 57 36" src="https://user-images.githubusercontent.com/76889135/154076743-6b3cf353-2726-4bb0-91c4-60e2ea26a48b.png">
i am new to vscode used to code in pycharm 
`solidity ^0.6.6`
and set vscode global compiler to 0.6.6 also

**Comment by n4n0b1t3** [comment]
please refer to https://github.com/smartcontractkit/full-blockchain-solidity-course-py/blob/main/chronological-issues-from-video.md#linting-issues

**Reply by ofirbenatar** [reply]
already look at it didnt find an answer to my problem

**Comment by n4n0b1t3** [comment, marked as answer]
I quickly wrote this post. Please try if this works for you.
https://github.com/smartcontractkit/full-blockchain-solidity-course-py/discussions/1069

**Reply by n4n0b1t3** [reply]
@cromewar We might consider rewriting the chronological issues from video file. This might work on Mac but did not for me on Windows, and I am quite sure, some people will just copy the settings code without realizing, that it needs to be adapted.
https://n4n0b1t3.medium.com/how-to-make-vsc-solidity-lint-recognize-your-chainlink-and-openzeppelin-libraries-73775129261c
Btw. the medium editor is absolutely awful, isn't it?

**Reply by cromewar** [reply]
@n4n0b1t3 Done ;)
And, yes that editor can be a little bit weird sometimes.
